website/docs/guides/version-3-upgrade.html.md

```diff
@@ -209,8 +213,27 @@ output "lambda_result" {
 }
 ```
 
+## Data Source: aws_route53_resolver_rule
+
+### Removal of trailing period in domain_name argument
+
+Previously the data-source returned the Resolver Rule Domain Name directly from the API, which included a `.` suffix. This proves difficult when many other AWS services do not accept this trailing period (e.g. ACM Certificate). This period is now automatically removed. For example, when the attribute would previously return a Resolver Rule Domain Name such as `example.com.`, the attribute now will be returned as `example.com`.
+While the returned value will omit the trailing period, use of configurations with trailing periods will not be interrupted.
+
+## Data Source: aws_route53_zone
+
+### Removal of trailing period in name argument
+
+Previously the data-source returned the Hosted Zone Domain Name directly from the API, which included a `.` suffix. This proves difficult when many other AWS services do not accept this trailing period (e.g. ACM Certificate). This period is now automatically removed. For example, when the attribute would previously return a Hosted Zone Domain Name such as `example.com.`, the attribute now will be returned as `example.com`.
+While the returned value will omit the trailing period, use of configurations with trailing periods will not be interrupted.
+
 ## Resource: aws_acm_certificate
 
+### Removal of trailing period in domain_name argument and domain_validation_options.domain_name attribute 
```

Comment:
This section can be removed as this should have been the previous API and resource behavior. 👍